source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -61,6 +61,26 @@ public new bool Exists()
 		public bool HasDonkHydraulics => Game.Version >= GameVersion.v1_0_505_2_Steam && SHVDN.NativeMemory.HasVehicleFlag(Model.Hash, NativeMemory.VehicleFlag2.HasLowriderDonkHydraulics);
 		public bool HasParachute => Game.Version >= GameVersion.v1_0_505_2_Steam && Function.Call<bool>(Hash._GET_VEHICLE_HAS_PARACHUTE, Handle);
 		public bool HasRocketBoost => Game.Version >= GameVersion.v1_0_944_2_Steam && Function.Call<bool>(Hash._GET_HAS_ROCKET_BOOST, Handle);
+		public bool IsRocketBoostActive
+		{
+			get => Game.Version >= GameVersion.v1_0_944_2_Steam && Function.Call<bool>((Hash)0x3D34E80EED4AE3BE, Handle);
+			set
+			{
+				if (Game.Version < GameVersion.v1_0_944_2_Steam)
+					throw new GameVersionNotSupportedException(GameVersion.v1_0_944_2_Steam, nameof(Vehicle), nameof(IsRocketBoostActive));
+				Function.Call((Hash)0x81E1552E35DC3839, Handle, value);
+			}
+		}
+		public bool IsParachuteActive
+		{
+			get => Game.Version >= GameVersion.v1_0_1011_1_Steam && Function.Call<bool>((Hash)0x3DE51E9C80B116CF, Handle);
+			set
+			{
+				if (Game.Version < GameVersion.v1_0_944_2_Steam)
+					throw new GameVersionNotSupportedException(GameVersion.v1_0_944_2_Steam, nameof(Vehicle), nameof(IsParachuteActive));
+				Function.Call((Hash)0x0BFFB028B3DD0A97, Handle, value);
```

Comment:
I'm sorry for not realizing what the 2nd paramater means until now, but that determines if the game allows the player to cancel parachuting (named `AllowPlayerCancelParachuting` in one of the files of the recent leak). We should separate the setter like `void StartParachuting(bool allowPlayerToCancelParachuting)`.

Reply:
No problem. which region should the method be placed?

Reply:
I struggled to think which region the method should be placed in, and I would say the method should look good in the new region `Special Abilities`.